_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+				),
+				checked: getSetting( state, 'related-posts' ),
+				optionsLink: '#/settings?term=related%20posts',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'related-posts': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	scan: {
+		mapStateToProps: state => {
+			const vaultPressData = getVaultPressData( state );
+			const isScanEnabled =
+				true === get( vaultPressData, [ 'data', 'features', 'security' ], false );
+
+			const sitePlan = getSitePlan( state );
+			const planClass = getPlanClass( sitePlan.product_slug );
+
+			const inCurrentPlan = [ 'is-premium-plan', 'is-business-plan', 'is-scan-plan' ].includes(
+				planClass
+			);
+
+			let optionsLink;
+			if ( inCurrentPlan ) {
+				optionsLink = '#/settings?term=scan';
+			}
+
+			let upgradeLink;
+			if ( ! inCurrentPlan ) {
+				upgradeLink = '#/plans';
+			}
+
+			let info;
+			if ( inCurrentPlan ) {
+				info = __( `Included with ${ sitePlan.product_name }` );
+			}
+
+			return {
+				title: __( 'Security scanning' ),
+				details: __( 'Stop threats to keep your website safe.' ),
+				checked: isScanEnabled,
+				isDisabled: inCurrentPlan,
+				isPaid: true,
+				optionsLink,
+				upgradeLink,
+				info,
+			};
+		},
+	},
+
+	search: {
+		mapStateToProps: state => {
+			let upgradeLink;
+			let optionsLink;
+			const sitePlan = getSitePlan( state );
+			if (
+				'is-business-plan' !== getPlanClass( sitePlan.product_slug ) &&
+				! hasActiveSearchPurchase( state )
+			) {
+				upgradeLink = '#/plans';
+			} else {
+				optionsLink = '/wp-admin/admin.php?page=jetpack#/settings?term=search';
+			}
+
+			return {
+				title: __( 'Search' ),
+				details: __(
+					'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.'
+				),
+				checked: getSetting( state, 'search' ),
+				isPaid: true,
+				upgradeLink,
+				optionsLink,
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { search: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	sso: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Secure Sign On' ),
+				details: __( 'Add an extra layer of security.' ),
+				checked: getSetting( state, 'sso' ),
+				optionsLink: '#/settings?term=secure%20sign%20on',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { sso: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	seo: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'SEO' ),
+				details: __( 'Take control of the way search engines represent your site.' ),
+				checked: getSetting( state, 'seo-tools' ),
+				configureLink: '#/settings?term=seo',
```

Comment:
This is a paid module.

Reply:
Good catch! I updated it to reflect this.